src/components/SearchInput/SearchInput.tsx

```diff
@@ -8,11 +9,21 @@ type Props = {
   value: string;
   setValue: (e: any) => void;
   onKeyDown: (e: any) => void;
+  onInput?: (e: FormEvent<HTMLInputElement>) => void;
```

Comment:
i guess i've previously suggested to use onInput instead of noChange. i've realized that in react onChange works the same as native "input" event. so onChange should be sufficient, sorry for confusion

Reply:
It turns out `onInput` is not exactly the same as `onChange`, as the latter one is not triggered when we paste a text into the input field. To solve this either `onPaste` callback should be used along with `onChange` (in this case the handler will be the same), or alternatively one can set `onInput` callback instead, which is triggered both on text change and on paste.